cocoapodsinstaller_test.go

```diff
@@ -0,0 +1,150 @@
+package main
```

Comment:
nit: the filename should be `cocoapods_installer_test.go`

Reply:
updated in https://github.com/bitrise-steplib/steps-cocoapods-install/pull/90/commits/91677485c76f0d1ac0e335809b20b54531d99ec2